src/desktop/src/ui/components/Dropzone.js

```diff
@@ -134,7 +134,8 @@ class Dropzone extends React.Component {
                 <h5 onClick={this.open}>
                     <Icon icon="seedVault" size={48} />{' '}
                     <span>
-                        {t('seedVault:')} Drop SeedVault file here<br /> or click to browse
+                        {t('seedVault:dropSeedVaultHere')} <br/>
+                        {t('seedVault:orClickToBrowse')}
```

Comment:
Instead of breaking the strings apart, it's best if we simply add the `<br />` to the translation string and let the translators put it in